src/Commands/MpxImporter.php

```diff
@@ -149,7 +149,7 @@ public function warmCache(string $media_type_id) {
 
     $factory = $this->dataObjectFactoryCreator->fromMediaSource($media_source);
     $query = new ObjectListQuery();
-    $request = $factory->selectRequest($query, $account);
+    $request = $factory->selectRequest($query);
```

Comment:
Looks like `$account` is unused in this method now. Remove definition on line 148?